livekit-agents/livekit/agents/voice/agent_activity.py

```diff
@@ -1638,10 +1641,14 @@ async def _read_text() -> AsyncIterable[str]:
         started_speaking_at: float | None = None
         stopped_speaking_at: float | None = None
 
-        def _on_first_frame(_: asyncio.Future[None]) -> None:
+        def _on_first_frame(fut: asyncio.Future[float] | asyncio.Future[None]) -> None:
```

Comment:
This isn't obvious that the float inside the future was for the started_speaking_at